src/index.ts

```diff
@@ -0,0 +1,121 @@
+import { type RemotePinningServiceClient, type Pin, type PinStatus, type PinsRequestidPostRequest, Status } from '@ipfs-shipyard/pinning-service-client'
+import { multiaddr } from '@multiformats/multiaddr'
+import debug from 'debug'
+import pRetry from 'p-retry'
+import { FailedToConnectToDelegates } from './errors.js'
+import type { Helia } from '@helia/interface'
+import type { CID } from 'multiformats/cid'
+
+const log = debug('helia-remote-pinning')
```

Comment:
Suggestion for consistency:

```suggestion
const log = debug('helia:remote-pinning')
```

Also, consider using [@libp2p/logger](https://www.npmjs.com/package/@libp2p/logger) instead as it gives you separate error/trace loggers so you don't need to set them up yourself and is used everywhere else in the stack.